### ISSUE TITLE
Bug: Time display goes from 0:01:00 to 0:00:60

### ISSUE THREAD

**Issue by mikebell** [ExileNG/NGSpaceCompany#20]
Odd thing I noticed while waiting for Segments to cool down the time briefly goes from 0:01:00 to 0:00:60 instead of 0:00:59. Not sure if it's a display bug or something deeper.

**Comment by eliannelavoie** [comment]
Got it!

This project uses [NumeralJS](https://www.npmjs.com/package/numeral) and the problem is on their side:
https://github.com/adamwdraper/Numeral-js/issues/468

The project is also completely abandoned, so I doubt it'll ever be fixed.

There is also a fork of NumeralJS that was last updated 9 months ago, Numbro, but it also has the same bug. Plus, they say "if you have time formatting needs, you should probably use another library".

All this to say: we just need to replace the calls to NumeralJS when it's used to format time by something else.

I'd recommend not using a library and just using a date object:
```js
let date = new Date(0, 0, 0, 0, 0, 59.75); // Date object, 59.75 seconds from Epoch
let timeString = date.toTimeString(); // '00:00:59 GMT-0500 (GMT-05:00)'
timeString = timeString.split(' ')[0]; // "00:00:59"

function formatSeconds(s) {
  return new Date(0, 0, 0, 0, 0, s).toTimeString().split(' ')[0];
}

console.log(
  formatSeconds(59), // "00:00:59"
  formatSeconds(59.75), // "00:00:59"
  formatSeconds(60) // "00:01:00"
)
``

**Comment by ExileNG** [comment]
Fixed in 1.29.2
